### DISCUSSION TITLE
[Request] reinstant ban for ABJ4403

### DISCUSSION THREAD

**Discussion by GreenMystic** [community/community#142817] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

hi my friend [ABJ4403](http://github.com/ABJ4403)
has been shadow banned

can you lift the ban?
his ticket #2978735,#2928167
thanks 🙏

**Comment by LiteBrite82** [comment, marked as answer]
Hi @GreenMystic 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
